### PR TITLE
added: landing page

### PR DESCRIPTION
**Pull request by rkapoor10**
Landing Page
- Logo
- Message
- Main Button

- deploy preview https://deploy-preview-1--kwizzit.netlify.app/

quiz landing

### CODE REVIEW

File: landing/landing.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
+/* || IMAGE STYLES  */
+.hero-img {
+  margin-top: 5rem;
+  max-width: 15rem;
+  max-width: 15rem;
```
Comment (Namrata-J) [inline landing/landing.css:5]: you have repetitive css declaration declared here!!

File: landing/landing.css [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  transition: 0.5s;
+  background-size: 200% auto;
+  color: white;
+  box-shadow: 0 0 20px var(--background-color);
+  border-radius: 5px;
+  border: 2px solid transparent;
+}
+.btn-grad:hover {
+  background-position: right center;
+  color: #fff;
```
Comment (Namrata-J) [inline landing/landing.css:40]: try using color variables

**Review by Namrata-J — APPROVED** [review]

File: landing/landing.css (hunk 1 above)
Comment (rkapoor10) [reply landing/landing.css:5]: done

File: landing/landing.css (hunk 2 above)
Comment (rkapoor10) [reply landing/landing.css:40]: made changes

**Review by mariyasada — APPROVED** [review]
looks good

**Pull request merged**